Modules/ReportGenerator/Database/Migrations/2018_06_04_083903_create_draggable_component_report_format_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
I assume these names are for WIP versioning for now, or are they intended to serve in a meaningful way to version components later?

Reply:
@aethelwulffe that's the laravel convention. I am trying to makek the table names as obvious as possible.